src/main/java/edu/bu/metcs673/trackr/bankaccount/BankAccountController.java

```diff
@@ -33,78 +31,101 @@
 @Validated
 @RestController
 @RequestMapping("/api/v1/bank-accounts")
-public class BankAccountController {
+public class BankAccountController extends CommonDataUtil {
```

Comment:
I have looked at `CommonDataUtil` and looks like the proper name for it is the following. BankAccountController `is-a` controller which extends a base controller. Making the base controller abstract really indicate it doesn't do anything on its own meant to b extended for by a concrete implementation. `CommonDataUtil` sends the wrong message that it's just an util class that we can add any anywhere. A controller `is-not` an util class.
```
java public abstract class AbstractController {
````

Reply:
Good idea! I have made it abstract and will call it `BaseController`